package.json

```diff
@@ -1,49 +1,49 @@
 {
   "name": "@salesforce/plugin-auth",
   "description": "plugin for sfdx auth commands",
-  "version": "2.8.26",
+  "version": "3.0.0",
   "author": "Salesforce",
   "bugs": "https://github.com/forcedotcom/cli/issues",
-  "main": "lib/index.js",
   "dependencies": {
-    "@oclif/core": "^2.15.0",
+    "@oclif/core": "^3.10.2",
     "@salesforce/core": "^5.3.17",
     "@salesforce/kit": "^3.0.15",
-    "@salesforce/sf-plugins-core": "^3.1.28",
+    "@salesforce/sf-plugins-core": "^4.0.0",
     "@salesforce/ts-types": "^2.0.9",
-    "chalk": "^4.1.2",
-    "inquirer": "^8.2.6",
-    "open": "^8.2.1",
-    "tslib": "^2"
+    "chalk": "^5.3.0",
+    "inquirer": "^9.2.11",
```

Comment:
this is using v9 inquirer with the v8 inquirer types.  That seems like a bug.  Willie had this problem on https://github.com/salesforcecli/plugin-sobject/pull/439#pullrequestreview-1710534742

Reply:
The problem is that sf-plugins-core is getting it's type definitions for inquirer from whatever version plugin-auth has. I think sf-plugins-core needs to include inquirer's types as a dependency so that this doesn't happen